TestProjects/UniversalGraphicsTest_Terrain/Assets/Scenes/140_SSAO/Renderers/SSAO_DeferredRenderer.asset

```diff
@@ -41,8 +40,8 @@ MonoBehaviour:
     failOperation: 0
     zFailOperation: 0
   m_ShadowTransparentReceive: 1
-  m_RenderingMode: 1
```

Comment:
Is this correct? Seems like m_RenderingMode = 0 is changing to forward renderer?